assets/stylesheets/common/wizard-admin.scss

```diff
@@ -707,3 +707,26 @@
   width: 80px;
   vertical-align: middle;
 }
+
+.btn-pavilion-pro {
+  background: #3c1c8c;
```

Comment:
@keegangeorge We should set this purple as a variable in a new css file called ``wizard-variables.scss``.